src/lib.rs

```diff
@@ -161,6 +161,14 @@ impl<T: Clone + Signed> Complex<T> {
 
 #[cfg(any(feature = "std", feature = "libm"))]
 impl<T: Float> Complex<T> {
+    /// Create a new Complex with a given phase -- `exp(i * phase)`
+    ///
+    /// [cis (mathematics)]: https://en.wikipedia.org/wiki/Cis_(mathematics)
+    #[inline]
+    pub fn cis(phase: T) -> Self {
+        (Self::i() * phase).exp()
```

Comment:
While this is mathematically fine, it would be more direct to create it directly:

```suggestion
        Self::new(phase.cos(), phase.sin())
```

Reply:
I was on the fence about the literal exponent initially; I'm more than happy to do it this way.